lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -22,10 +22,11 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
 
 <script src="./dbw_tester.js"></script>
-<link rel="stylesheet" type="text/css" href="./dbw_tester.css">
+<link rel="stylesheet" href="./dbw_tester.css">
+<link rel="stylesheet" href="./dbw_disabled.css" disabled>
 <style>
   body {
-    color: red;
+    color: white;
```

Comment:
Ah, just b/c the text is no longer visible on the text page. It clashes with the new stylesheet.